### DISCUSSION TITLE
Only 1 inverter found

### DISCUSSION THREAD

**Discussion by lhaperen** [wlcrs/huawei_solar#196] [Setting up the integration]
I have 3 Inverters which are lined together with the RS484 cable and the total installation is connected to the internet (LAN) by the Sdongle. But when I try to add the Inverters HA sees only one of them. Therefore Modbus on the dongle seems to wordt but why cant i see the other two inverters not. Or is this not posisble with this intergration?

**Comment by wlcrs** [comment]
Have you followed the instructions on https://github.com/wlcrs/huawei_solar/wiki/Connecting-to-daisy-chained-inverters ?

**Reply by pmiola73** [reply]
Hi I have the same issue: 2 inverter in daisy chain, master has IDs 16, slave has Ids 1
If  try to configure the integration using this string in the slave ids field 16,1 I get an error. 
any ideas?

thanks

**Reply by ogghi** [reply]
Exact same issue here.
If I connect only to "1" it works, but I only get one inverters data...

**Comment by LotharWoman** [comment]


**Reply by ogghi** [reply]
I once was able to connect to ID 1. Then I was not able to connect to other IDs (I did not have the IDs yet).
Then I removed the addon, removed the folder under HACS folder.
Added again (always restarts in between). Now I still can't add **anything** until now.
Tonight I'll try and restart the inverters perhaps?

**Reply by ogghi** [reply]
Ha, got it to run:\
Removed integration, restarted HA. Installed integration, restarted HA.
Added with 16,1 in the ID field, then all at a sudden it added all devices: Battery, Smart Meter, Inverter 1, Inverter 2
